components/camel-sjms/pom.xml

```diff
@@ -1,36 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-
```

Comment:
Please remove all the extra lines added to keep the changes as small as possible

Reply:
Thanks for notice, now I commited this only as a draft, soon it will be marked in this way. I'm still working on this ticket